src/rtctools_heat_network/electricity_physics_mixin.py

```diff
@@ -245,6 +249,67 @@ def __electricity_cable_heat_mixing_path_constraints(self, ensemble_member):
 
         return constraints
 
+    def __voltage_loss_path_constraints(self, ensemble_member):
+        """
+        Furthermore, the voltage_loss symbol is set, as it depends on the chosen pipe
+        class, e.g. the related resistance and the current through the cable.
+
+        Parameters
+        ----------
+        ensemble_member : The ensemble of the optimization
+
+        Returns
+        -------
+        list of the added constraints
+        """
+        constraints = []
+        parameters = self.parameters(ensemble_member)
+
+        for cable in self.heat_network_components.get("electricity_cable", []):
+            cable_classes = []
+
+            current = self.state(f"{cable}.ElectricityIn.I")
+            v_loss = self.state(f"{cable}.V_loss")
+            r = parameters[f"{cable}.r"]
+            # v_loss_nom = parameters[f"{cable}.nominal_voltage_loss"]
+            v_nom = parameters[f"{cable}.nominal_voltage"]
+            c_length = parameters[f"{cable}.length"]
+
+            constraint_nominal = self.variable_nominal(v_loss)
+
+            # TODO: still have to check for proper scaling
+            try:
```

Comment:
I would prefer if we instead of a try except check if the cable is in the topo maps and have an if/else structure. This to have a bit more predictable behaviour of the code

Reply:
Agreed, I will change it